### PR TITLE
Remove fragment, presenter and view

### PR DESCRIPTION
**Pull request by Jimenarosello**
**Summary** 
Remove presenter and fragment for session management.

### CODE REVIEW

File: app/src/main/java/ar/com/wolox/android/example/ui/session/SessionActivity.java [hunk 1]
```diff
@@ -1,19 +1,33 @@
...
 import ar.com.wolox.wolmo.core.activity.WolmoActivity;
+import kotlin.text.StringsKt;
 
 /**
  *  Session activity. It will be in charge of managing the logged user's info.
  */
 public class SessionActivity extends WolmoActivity {
+
+    @Inject
+    public UserSession userSession;
```
Comment (Jimenarosello) [inline app/src/main/java/ar/com/wolox/android/example/ui/session/SessionActivity.java:17]: Is there a better way to do this? Dagger doesn't let me inject a private field 😢

File: app/src/main/java/ar/com/wolox/android/example/ui/session/SessionActivity.java [hunk 2]
```diff
@@ -1,19 +1,33 @@
...
+
     @Override
     protected int layout() {
         return R.layout.activity_base;
     }
 
     @Override
     protected void init() {
-        replaceFragment(R.id.vActivityBaseContent, SessionFragment.newInstance());
+        if (StringsKt.isBlank(userSession.getUsername())) {
```
Comment (igsosa92) [inline app/src/main/java/ar/com/wolox/android/example/ui/session/SessionActivity.java]: You should add a function in userSession so you can ask `userSession.isOnGoingSession` or `userSession.isUserLogged` and delegate that login to userSession

File: app/src/main/java/ar/com/wolox/android/example/ui/session/SessionActivity.java (hunk 1 above)
Comment (igsosa92) [reply app/src/main/java/ar/com/wolox/android/example/ui/session/SessionActivity.java:17]: 👀

**Review by igsosa92 — CHANGES_REQUESTED** [review]

**Review by igsosa92 — APPROVED** [review]
![alt text][logo]

[logo]: https://media.giphy.com/media/mWW03bCGAIfPo9pKkP/giphy.gif

**Pull request merged**